accepted/single-file/extract.md

```diff
@@ -78,11 +78,14 @@ For a single-file app, the extraction directory is `<base>/<app>/<bundle-id>`
 * `<base>` is 
 
   * `DOTNET_BUNDLE_EXTRACT_BASE_DIR` environment variable, if set.
-  * If not, defaults to 
-    * `%TEMP%\.net ` on Windows
-    * `$TMPDIR/.net` if `$TMPDIR` is set (Posix conforming OSes including Mac)
-    * Otherwise `/var/tmp/.net` (Ubuntu)  if the directory exists.
-    * Otherwise  `/tmp/.net` 
+  * If not, on Windows, defaults to `%TEMP%\.net`
+  * On Unix-like systems, where multiple users may use a single system, an approach that removes the possibility of name collisions and other users creating files to prevent an application to start (by a malicious user creating a predictable directory name) is used instead:
```

Comment:
For Unix extraction directory, I think we should clarify that:
* In .net 3, the extraction directory is:
    * `$TMPDIR/<user-ID>/.net` if `$TMPDIR` is set. If not, 
    * `/var/tmp/<user-ID>/.net` if that exists, otherwise 
    * `/tmp/<user-ID>/.net` if that exists 
* In .net 5, the proposed new scheme is ...

Reply:
Done